enterprise/core-edge/src/main/java/org/neo4j/coreedge/raft/replication/tx/ReplicatedTransactionCommitProcess.java

```diff
@@ -111,15 +114,13 @@ transaction content does eventually get replicated (e.g. delayed on the network)
             }
             catch ( InterruptedException | TimeoutException  e )
```

Comment:
We cannot give up when interrupted, but we should save the interrupt state for later, and set it when returning.